libraries/Microsoft.Bot.Builder.AI.LUIS/Schemas/Microsoft.LuisRecognizer.schema

```diff
@@ -16,6 +16,11 @@
             "title": "LUIS Application ID",
             "description": "Application ID for your model from the LUIS service."
         },
+        "version": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "LUIS Version",
+            "description": "Optional version to target.  If null then predictionOptions.Slot is used."
+        },
         "endpoint": {
```

Comment:
I still don't like promoting this to the top given the presence in prediction options.

Reply:
this was not supposed to be in this PR

---
In reply to: [481305906](https://github.com/microsoft/botbuilder-dotnet/pull/4537#discussion_r481305906) [](ancestors = 481305906)